reflex/components/radix/themes/components/alert_dialog.py

```diff
@@ -48,18 +40,14 @@ class AlertDialogContent(elements.Div, RadixThemesComponent):
     # Whether to force mount the content on open.
     force_mount: Var[bool]
 
-    def get_event_triggers(self) -> Dict[str, Any]:
-        """Get the events triggers signatures for the component.
-
-        Returns:
-            The signatures of the event triggers.
-        """
-        return {
-            **super().get_event_triggers(),
-            EventTriggers.ON_OPEN_AUTO_FOCUS: lambda e0: [e0],
-            EventTriggers.ON_CLOSE_AUTO_FOCUS: lambda e0: [e0],
-            EventTriggers.ON_ESCAPE_KEY_DOWN: lambda e0: [e0],
-        }
+    # Fired when the dialog is opened.
+    on_open_auto_focus: EventHandler[lambda e0: [e0]]
+
+    # Fired when the dialog is closed.
+    on_close_auto_focus: EventHandler[lambda e0: [e0]]
+
+    # Fired when the escape key is pressed.
+    on_escape_key_down: EventHandler[lambda e0: [e0]]
```

Comment:
i wonder if we should mention that the intended purpose of these event handlers is primarily to call `rx.prevent_default`